lib/v2/scitation/radar.js

```diff
@@ -1,11 +1,11 @@
 module.exports = {
-    'scitation.org': {
+    'pubs.aip.org': {
```

Comment:
Top level object key can not match subdomain.

Reply:
Oh, I see. The modification should be as follows.